docs/src/proposals/append-vec-storage.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Reduce the size of AppendVec storage in AccountsDB
+---
+
+## Problems
+The storage format, StoredAccountMeta, of each entry in the AppendVec currently
+has 129 bytes overhead + the size of account data for each account.
+
+StoredAccountMeta currently has the following format:
+
+```
+pub struct StoredAccountMeta<'a> {
+    pub meta: &'a StoredMeta {
+        pub write_version: StoredMetaWriteVersion, // 8 bytes
+        pub pubkey: Pubkey,  // 32 bytes, encoded from a 44-byte string.
+        pub data_len: u64,   //  8 bytes
+    },
+    pub account_meta: &'a AccountMeta {
+        pub lamports: u64,     //  8 bytes
+        pub owner: Pubkey,     // 32 bytes, encoded from a 44-byte string
```

Comment:
This is not encoded from a string. This is the representation (Pubkey). It can be displayed as a string. And, the Pubkey can be transformed to/from a string.